src/parser.ts

```diff
@@ -72,6 +79,18 @@ export const getParser = (originalParse: Parser["parse"], parserName: string) =>
 
       const parsed = parse(commentString, {
         spacing: "preserve",
+        tokenizers: [
+          tagTokenizer(),
+          (spec) => {
+            if ([DEFAULT, DEFAULT_VALUE].includes(spec.tag)) {
```

Comment:
We can use the `isDefaultTag` util here instead.

Reply:
I will merge this, we could do this in another PR

Reply:
Do the tests still pass?

Reply:
yes Passed